Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int[] arr = new int[]{1, 2, 3, 4, 5};
+        System.out.println("Even numbers:");
+
+        for (int i = 0; i < arr.length; i = i * 2)
+        {
+            System.out.println(arr[i]);
+        }
+    }

```

Comment:
**Suggestion**: Consider renaming the method from Even to printEvenIndexedElements to follow Java naming conventions and improve readability. Refactor the method to eliminate code duplication, specifically the Scanner initialization and array declaration which are already present in the main method. Optimize the loop condition to ensure it correctly processes even indices without risking an ArrayIndexOutOfBoundsException. <br> **Code Suggestion**: 
 ```
 -    public static void Even(String[] args) {
 +    public static void printEvenIndexedElements(int[] arr) {
 +        System.out.println("Even numbers:");
 +        for (int i = 1; i < arr.length; i += 2)
 ```